File: Data Structures & Algorithms/single-number/submission-0.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {number}
     */
    singleNumber(nums) {
     let i = 0;
     let res = 0;
     while(i < nums.length) {
        res^=nums[i];
        i++;
     }
     return res;
    }
}

File: Data Structures & Algorithms/products-of-array-discluding-self/submission-0.py
class Solution:
    def productExceptSelf(self, nums: List[int]) -> List[int]:
        res = len(nums) * [1]
        for i in range(len(nums)):
            product = 1
            for j in nums[:i]:
                product = product * j
            for j in nums[i+1:]:
                product = product * j
            res[i] = product
        return res
